aws/cloudformation/components/www_redirect.yml.erb

```diff
@@ -21,16 +21,10 @@
               OriginProtocolPolicy: match-viewer
         DefaultCacheBehavior:
           TargetOriginId: PrimaryOrigin
-          LambdaFunctionAssociations:
-            # Trigging upon origin-request ensures redirect is cached
-            - EventType: origin-request
-              LambdaFunctionARN: !Ref <%=app%>WwwRedirectLambdaVersion1
-          ForwardedValues:
-            QueryString: "false"
-            Headers:
-              - Origin
-            Cookies:
-              Forward: none
+          CachePolicyId: !Ref <%=app%>RedirectCachePolicy
+          FunctionAssociations:
+            - EventType: viewer-request
+              FunctionARN: !Ref <%=app%>RedirectFunction
```

Comment:
were these missing `<%=app%>`s the only issue with https://github.com/code-dot-org/code-dot-org/pull/44410?

Reply:
yes, I was trying to **REF**erence resources that didn't exist, because I had not named them the same.